code/_onclick/hud/blob_overmind.dm

```diff
@@ -79,6 +93,14 @@
 	name = "Readapt Chemical (40)"
 	desc = "Randomly rerolls your chemical for 40 points."
 
+/obj/screen/blob/ReadaptChemical/MouseEntered(location,control,params)
+	if(isovermind(usr))
```

Comment:
short of *really intense* badminning and or shitcoding I'm pretty sure these checks aren't actually needed